libbroker/broker/internal/peering.cc

```diff
@@ -94,9 +94,12 @@ class suffix_generator : public affix_generator {
 
   node_message first() override {
     if (ptr_->removed()) {
+      std::string msg{"removed connection to remote peer"};
+      std::string reason = ptr_->removed_reason();
+      if (!reason.empty())
+        msg += " (" + reason + ")";
```

Comment:
This creates a lot of unnecessary temporaries (also should use `auto`).

```suggestion
      auto msg = "removed connection to remote peer"s;
      if (const auto& reason = ptr_->removed_reason(); !reason.empty()) {
        msg += " (";
        msg += reason;
        msg += ')';
      }
```